### ISSUE TITLE
[bug] Error when running new app after `npm create tamagui-app@latest`

### ISSUE THREAD

**Issue by kyle-frntlne** [tamagui/tamagui#377]
**Describe the bug**

Get below error after running `yarn native` and opening on iOS simulator after creating new project with `npm create tamagui-app@latest`

```
 ERROR  Error: NextRouter was not mounted. https://nextjs.org/docs/messages/next-router-not-mounted

This error is located at:
    in UserDetailScreen (created by SceneView)
    in StaticContainer
    in EnsureSingleNavigator (created by SceneView)
    in SceneView (created by SceneView)
    in RCTView (created by View)
    in View (created by DebugContainer)
    in DebugContainer (created by MaybeNestedStack)
    in MaybeNestedStack (created by SceneView)
    in RNSScreen (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by Screen)
    in MaybeFreeze (created by Screen)
    in Screen (created by SceneView)
    in SceneView (created by NativeStackViewInner)
    in RNSScreenStack (created by ScreenStack)
    in ScreenStack (created by NativeStackViewInner)
    in NativeStackViewInner (created by NativeStackView)
    in RNCSafeAreaProvider (created by SafeAreaProvider)
    in SafeAreaProvider (created by SafeAreaInsetsContext)
    in SafeAreaProviderCompat (created by NativeStackView)
    in NativeStackView (created by NativeStackNavigator)
    in Unknown (created by NativeStackNavigator)
    in NativeStackNavigator (created by NativeNavigation)
    in NativeNavigation (created by App)
    in EnsureSingleNavigator
    in BaseNavigationContainer
    in ThemeProvider
    in NavigationContainerInner (created by NavigationProvider)
    in NavigationProvider (created by Provider)
    in PortalProviderComponent
    in Theme2 (created by _ThemeProvider)
    in _ThemeProvider (created by _TamaguiProvider)
    in _TamaguiProvider
    in Unknown (created by Provider)
    in Provider (created by App)
    in App (created by ExpoRoot)
    in ExpoRoot
    in RCTView (created by View)
    in View (created by AppContainer)
    in DevAppContainer (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer
    in main(RootComponent)
```

**To Reproduce**

```
npm create tamagui-app@latest
cd myapp
yarn native
```

**System Info (please complete the following information):** 
```
  System:
    OS: macOS 12.6
    CPU: (8) x64 Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
    Memory: 1.14 GB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 16.14.2 - ~/.nvm/versions/node/v16.14.2/bin/node
    Yarn: 3.2.3 - ~/.nvm/versions/node/v16.14.2/bin/yarn
    npm: 8.19.1 - ~/.nvm/versions/node/v16.14.2/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Browsers:
    Chrome: 107.0.5304.110
    Safari: 16.0
  npmPackages:
    @babel/runtime: ^7.18.9 => 7.18.9
    @manypkg/cli: ^0.19.1 => 0.19.1
    @types/react: ^18.0.15 => 17.0.21
    @types/react-native: ^0.69.6 => 0.69.15
    eslint: ^8.21.0 => 8.21.0
    prettier: ^2.7.1 => 2.7.1
    turbo: ^1.4.2 => 1.4.2
    typescript: ^4.7.4 => 4.7.4
```

**Comment by kyle-frntlne** [comment]
It looks like the issue is related to Solito's `useParam`

```
const { useParam } = createParam<{ id: string }>()
...
const [id] = useParam('id')
```

**Comment by nandorojo** [comment]
I think I know what’s up. I’ll release an update for solito tomorrow with a fix.

**Comment by nandorojo** [comment]
Should be fixed with `solito` version `2.0.0-canary.4`.

```sh
yarn add solito@next13
```

**Comment by natew** [comment]
Pushed this fix to master and tested it's working
